### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ivy-sgi**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
+ - Device: [e.g. iPhone6]
+ - OS: [e.g. iOS8.1]
+ - Browser [e.g. stock browser, safari]
+ - Version [e.g. 22]
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (phochstr) [inline .github/ISSUE_TEMPLATE/bug_report.md:40]: I'd add add here:

- if you get error messages, please include the ivy logs showing these errors.
- if you experience a stuck application, please create several thread dumps.

Can we create a page containing a checklist what to include and procedures, like for instance creating thread dumps with jstack etc. ?

**Review by phochstr — CHANGES_REQUESTED** [review]
Even though this is open source, it is using Ivy. So, we have to gather as much information as possible so we or others can analyze what happens. Logs and thread dumps are the first two that come to my mind, but there are certainly others.

Therefore, I'd opt for a link to a checklist and procedures library.

**Comment by alexsuter** [comment]
I'll close the PR and prefix the branch with stale/*. So that there are no builds running on our Jenkins server.

If you want to continue the work you can create a PR out of the stale/* branch.

The demo-projects is a special Axon Ivy Market artifact which can only be successfully built on our Jenkins server (because of test infrastructure which is only available in our internal network). For each branch builds are running unless they are prefix with stale/*.

**Comment by alexsuter** [comment]
I'll close the PR and prefix the branch with stale/*. So that there are no builds running on our Jenkins server.

If you want to continue the work you can create a PR out of the stale/* branch.

The demo-projects is a special Axon Ivy Market artifact which can only be successfully built on our Jenkins server (because of test infrastructure which is only available in our internal network). For each branch builds are running unless they are prefix with stale/*.

**Pull request closed without merging**